### DISCUSSION TITLE
RSS intergration?

### DISCUSSION THREAD

**Discussion by Armo00** [yagiziskirik/Kerbal-Telemetry#15] [Ideas]
This mod looks very cool and I would like to have it in my RSS KSP. Is it possible to do that?
Also, how can I help if I wish to contribute to this mod? I do have a little experience on JavaScript and basic web design.

**Comment by yagiziskirik** [comment]
Hi @Armo00,

First of all, thanks for the compliment! I tried my best to make it look cool 😎 I received a similar question about RSS and since I am moving, I couldn't test it out yet. I will install RSS and try to use KT with it soon.

For the helping part, the website design is straightforward but all over the place right now. I am trying to make it more universal and accessible from every device. I started this project as a prototype; sadly, it still looks like a prototype on the coding side. I will rewrite the entire website interface from scratch and your help would be awesome! I can say that you have brilliant ideas about the Kerbal Telemetry and I am looking forward to working with you. Send me an email to this address if you are interested: yagiziskirik@icloud.com!

**Reply by Armo00** [reply]
Hi there, I have sent you the email about 9 days ago and I have not received any response. I am wondering if my email reached you or not. Anyway, I am really interested in this project and I am looking forward to hearing from you!
